ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/MigrationConfig.java

```diff
@@ -0,0 +1,139 @@
+import io.confluent.ksql.rest.entity.ServerInfo;
+import io.confluent.ksql.util.KsqlException;
+import java.io.File;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+import org.apache.kafka.common.config.AbstractConfig;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.common.config.ConfigDef.Importance;
+import org.apache.kafka.common.config.ConfigDef.Type;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MigrationConfig extends AbstractConfig {
+
+  public static final String KSQL_SERVER_URL = "ksql.server.url";
+
+  public static final String KSQL_MIGRATIONS_STREAM_NAME = "ksql.migrations.stream.name";
+  public static final String KSQL_MIGRATIONS_STREAM_NAME_DEFAULT = "migration_events";
+  public static final String KSQL_MIGRATIONS_TABLE_NAME = "ksql.migrations.table.name";
+  public static final String KSQL_MIGRATIONS_TABLE_NAME_DEFAULT = "schema_version";
+  public static final String KSQL_MIGRATIONS_STREAM_TOPIC_NAME =
+      "ksql.migrations.stream.topic.name";
+  public static final String KSQL_MIGRATIONS_TABLE_TOPIC_NAME = "ksql.migrations.table.topic.name";
+  public static final String KSQL_MIGRATIONS_TOPIC_REPLICAS = "ksql.migrations.topic.replicas";
+  public static final int KSQL_MIGRATIONS_TOPIC_REPLICAS_DEFAULT = 1;
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(MigrationConfig.class);
+
+  public static MigrationConfig load() {
+    try {
+      final Map<String, String> configsMap =
+          PropertiesUtil.loadProperties(new File("ksql-migrations.properties"));
+      return new MigrationConfig(configsMap, getServiceId(configsMap));
+    } catch (KsqlException e) {
+      LOGGER.error(e.getMessage());
+      System.exit(1);
+    }
+    return null;
+  }
+
+  private MigrationConfig(final Map<String, String> configs, final String id) {
+    super(new ConfigDef()
+        .define(
+            KSQL_SERVER_URL,
+            Type.STRING,
+            "",
+            Importance.HIGH,
+            "The URL for the KSQL server"
+        )
+        .define(
+            KSQL_MIGRATIONS_STREAM_NAME,
+            Type.STRING,
+            KSQL_MIGRATIONS_STREAM_NAME_DEFAULT,
+            Importance.MEDIUM,
+            "The name of the migration stream. It defaults to "
+                + KSQL_MIGRATIONS_STREAM_NAME_DEFAULT
+        ).define(
+            KSQL_MIGRATIONS_TABLE_NAME,
+            Type.STRING,
+            KSQL_MIGRATIONS_TABLE_NAME_DEFAULT,
+            Importance.MEDIUM,
+            "The name of the migration table. It defaults to "
+                + KSQL_MIGRATIONS_TABLE_NAME_DEFAULT
+        ).define(
+            KSQL_MIGRATIONS_STREAM_TOPIC_NAME,
+            Type.STRING,
+            id + "ksql_" + configs.getOrDefault(KSQL_MIGRATIONS_STREAM_NAME, KSQL_MIGRATIONS_STREAM_NAME_DEFAULT),
+            Importance.MEDIUM,
+            "The name of the migration stream topic. It defaults to "
```

Comment:
This description is incomplete. Same for `KSQL_MIGRATIONS_TABLE_TOPIC_NAME` below.